crates/eframe/src/native/run.rs

```diff
@@ -414,21 +525,28 @@ mod glow_integration {
     /// The setup is divided between the `new` fn and `on_resume` fn. we can just assume that `on_resume` is a continuation of
     /// `new` fn on all platforms. only on android, do we get multiple resumed events because app can be suspended.
     struct GlutinWindowContext {
-        builder: winit::window::WindowBuilder,
         swap_interval: glutin::surface::SwapInterval,
         gl_config: glutin::config::Config,
+
         current_gl_context: Option<glutin::context::PossiblyCurrentContext>,
-        gl_surface: Option<glutin::surface::Surface<glutin::surface::WindowSurface>>,
         not_current_gl_context: Option<glutin::context::NotCurrentContext>,
-        window: Option<winit::window::Window>,
+
+        viewports: HashMap<ViewportId, Arc<RwLock<Window>>>,
+        viewports_maps: HashMap<winit::window::WindowId, ViewportId>,
+        builders: HashMap<ViewportId, ViewportBuilder>,
     }
 
+    #[allow(unsafe_code)]
+    unsafe impl Sync for GlutinWindowContext {}
+    #[allow(unsafe_code)]
```

Comment:
Please explain in a comment why this is safe!

Reply:
That is not safe because the gl context is not thread safe and neither clipboard!
That was a workaround to work on sync viewports when i didn't think sync viewport was possible! 

I did this commit to resolve that [449f38a](https://github.com/emilk/egui/pull/3172/commits/449f38adf2837c0e3663914f65aeaf595faa3139)

But this makes it impossible to have two different `egui::Context` on the same thread!
There is a way to store `std::thread::LocalKey` on a struct?